deploy.sh

```diff
@@ -57,7 +64,7 @@ elif [ "$1" == "gtest-setup" ]; then
 	sudo apt-get update
 	sudo apt-get install -y build-essential
 	sudo apt-get install -y cmake libgtest-dev
-	
+    sudo apt-get install -y gcovr	
```

Comment:
Tab to align with other commands. Consider merging all dependencies into a single `apt-get` command

Reply:
:doh: